Cargo.toml

```diff
@@ -2,27 +2,35 @@
 members = [
   # CI benchmarks
   "ci-bench",
-  # Tests that require OpenSSL
-  "openssl-tests",
   # Network-based tests
   "connect-tests",
   # tests and example code
   "examples",
-  # the main library and tests
-  "rustls",
+  # Tests that require OpenSSL
+  "openssl-tests",
   # example of custom provider
   "provider-example",
+  # the main library and tests
+  "rustls",
   # experimental post-quantum algorithm support
   "rustls-post-quantum",
+  # test certificates and keys for rustls unit tests
+  "test-ca",
```

Comment:
I started out by writing a single file Cargo script like the [`update_valid_ee_certs.rs`](https://github.com/rustls/rustls-platform-verifier/blob/main/rustls-platform-verifier/src/tests/verification_real_world/update_valid_ee_certs.rs) tool in the rustls-platform-verifier repo but ended up going back to a boring-old binary crate for a couple reasons:

1. There's no `cargo fmt` or `cargo clippy` support for these single file scripts at the current time, which is a huge downside IMO.
2. Running the script requires using nightly rust and that's a bit of a bummer.

Reply:
How about making this an `example` in one of the existing crates?

Reply:
I've moved the tool to `rustls/examples/internal`.